### ISSUE TITLE
OSI ModelReference String

### ISSUE THREAD

**Issue by jdsika** [OpenSimulationInterface/open-simulation-interface#835]
Would it be a sesitive next step to define the ModelReference String as URI?

**Comment by pmai** [comment]
I don't think it makes sense to do so, especially as long as no common definition of file formats has happened, and as different use cases have different requirements:

- In some use cases the model reference is just a key that the receiver will know what to do with (e.g. it has built-in models that the references apply to); while of course URIs can be invented for this purpose, they would just mislead other receivers into believing they have semantics that they don't.
- Even in cases where the sender would know how to locate the models via URIs, there are additional conventions that would need to be defined (i.e. what schemes are allowed, are relative URIs allowed/required), so that a receiver can then use the URIs reliably to locate the same models. And even then as long as there is no agreement on the file formats that should be supported, this still all boils down to individual agreements to make this work.

So as long as this is the case I do not see the point (i.e. the portability enhancement) that mandating URIs would bring. Of course anyone who wants to use URIs is allowed to do so anyways.

**Comment by ClemensLinnhoff** [comment]
I agree.
But I think this is something that we can define in OpenMATERIAL for example. So if an OSI message is used in combination with OpenMATERIAL compliant 3D assets, the model_reference shall have a certain form.

**Comment by ClemensLinnhoff** [comment]
I added an issue for this in OpenMATERIAL: https://github.com/asam-ev/OpenMATERIAL/issues/160
